readthedocs/rtd_tests/tests/test_api.py

```diff
@@ -140,11 +138,14 @@ def test_api_does_not_have_private_config_key_normal_user(self):
         self.assertNotIn('_config', resp.data)
 
     def test_save_same_config_using_patch(self):
-        client = APIClient()
-        client.login(username='super', password='test')
         project = Project.objects.get(pk=1)
         version = project.versions.first()
         build_one = Build.objects.create(project=project, version=version)
+
+        client = APIClient()
+        _, build_api_key = BuildAPIKey.objects.create_key(project)
+        client.credentials(HTTP_AUTHORIZATION=f"Token {build_api_key}")
```

Comment:
Should we make this part of the `setUp()` method instead of repeating it in all these tests?

Reply:
Each test has a different project attached to the key, and we test anonymous users too, so this will need to be repeated anyway.